core/types.ml

```diff
@@ -2046,7 +2046,7 @@ module Policy = struct
              | AllImplicitArrowsShare
     type t = opt list
 
-    let default_opts = [PresenceOmit ; AliasOmit ; ContractOperationArrows ; ArrowsCurriedHideFresh ; FinalArrowSharesWithAlias ]
+    let default_opts = [PresenceOmit ; AliasOmit ; ContractOperationArrows ; ArrowsCurriedHideFresh ]
```

Comment:
... and these are only toggled if `effect_sugar=true`, right?

Reply:
Yes. The setting itself is independent of `effect_sugar`, so the values are there, but they take no effect unless `effect_sugar=true`

Reply:
What I mean is that the `default_opts` will be set even without `effect_sugar=true`, but this will have no effect on anything.